### DISCUSSION TITLE
can zwavejs2mqtt discover device name and location from anything other than smartstart provisioning data?

### DISCUSSION THREAD

**Discussion by fallenpegasus** [zwave-js/zwave-js-ui#2314] [Ideas]
It would be useful if I could keep a spreadsheet of all my devices, with columns for id, location, and name, and then inload that into zwavejs2mqtt before i attach it to a new stick for an existing zwave network.  Is there such a thing, or is this a feature request?

**Comment by robertsLando** [comment]
@fallenpegasus This could be done using nodes.json file ATM but you should edit it manually
